README.md

```diff
@@ -22,11 +22,12 @@ Some use cases include:
 
 ## Usage
 
-* Install the package
+Install the package
+
 
     npm install --save express-interceptor
 
-* Define your interceptor
+Define your interceptor
```

Comment:
Is it possible to include comments in the code snippet? Maybe the interceptor could be defined with a variable name, and add more comments to the returned object saying what it does:

````
var addFooterInterceptor = interceptor(function(req, res) {
  return {
    // Only HTML responses will be intercepted
    isInterceptable: function() {},
    // Appends a paragraph at the end of the response body
    intercept: function(body, send) {}
  };
});
````